gitlab-service-reconection-required.rst

```diff
@@ -0,0 +1,24 @@
+.. post:: Sep 14, 2022
+   :tags: gitlab
+   :author: Santos
+   :location: CUE
+
+GitLab service re-connection required
+=====================================
+
+Some `months ago`_ GitLab started enforcing an expiration time of two hours for all of their OAuth tokens.
+
+.. _months ago: https://gitlab.com/gitlab-org/gitlab/-/blob/master/CHANGELOG.md#1500-2022-05-20
```

Comment:
their changelog takes some time to render... maybe just link to https://gitlab.com/gitlab-org/gitlab/-/merge_requests/86362?

Reply:
I think it's fine either way. Most people won't click the link.